### DISCUSSION TITLE
Debug statements?

### DISCUSSION THREAD

**Discussion by daryllabar** [Power-Platform-Patterns-and-Practices/Dataverse.Sdk.Extensions#4] [Ideas]
I've written some extensions for logging things to the plugin trace,  interested in including it?

https://github.com/daryllabar/DLaB.Xrm/blob/master/DLaB.Xrm.Core/Extensions/ToStringDebugExtensions.cs

**Comment by MscrmTools** [comment]
Definitively!

**Comment by daryllabar** [comment]
Done.
